platform/switch_ppc64_linux.h

```diff
@@ -56,7 +56,7 @@
 #define ALTIVEC_REGS
 #endif
 
-#define REGS_TO_SAVE "r2", "r14", "r15", "r16", "r17", "r18", "r19", "r20", \
+#define REGS_TO_SAVE "r14", "r15", "r16", "r17", "r18", "r19", "r20", \
        "r21", "r22", "r23", "r24", "r25", "r26", "r27", "r28", "r29", "r31", \
```

Comment:
Why is r30 missing?
It's also nonvolatile in the [ppc64le ABI](http://openpowerfoundation.org/wp-content/uploads/resources/leabi/content/dbdoclet.50655240_68174.html)

Reply:
Is it a left-over from reusing ppc code?
If it is, that is not valid for ppc64{le}.